setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup, find_packages
 
-version = '2.2.5'
+version = '2.2.5+zg1'
```

Comment:
Shouldn't this be `2.2.5+zg1.1`? It's our (ZG's`) 1.1 release?

Reply:
True, updated.